### PR TITLE
WIP OSD-5651: Changes to allow onboarding boilerplate

### PR DESCRIPTION
**Pull request by NautiluX**
* Changes to allow usage of openshift/boilerplate
* This PR depends [on this PR in boilerplate repo](https://github.com/openshift/boilerplate/pull/116)
* After the PR in boilerplate is merged, boilerplate needs to be bootstrapped in Route Monitor Operator as described in the [boilerplate README](https://github.com/openshift/boilerplate#quick-start)

### CODE REVIEW

**Comment by RiRa12621** [comment]
/retitle WIP OSD-5651: Changes to allow onboarding boilerplate

File: controllers/routemonitor/routemonitor_supplement.go [hunk 1]
```diff
@@ -53,16 +52,3 @@ func (r *RouteMonitorReconciler) EnsureRouteMonitorAndDependenciesAbsent(ctx con
 	}
 	return utilreconcile.StopReconcile()
 }
-
-// ensureServiceMonitoRelatedResourcesrAbsent assumes that the ServiceMonitor that is related was deleted
```
Comment (RiRa12621) [inline controllers/routemonitor/routemonitor_supplement.go:57]: Can you explain that bit?

Comment (NautiluX) [reply controllers/routemonitor/routemonitor_supplement.go:57]: unused, so it fails the build

File: Makefile [hunk 2]
```diff
@@ -6,6 +6,10 @@ CURRENT_COMMIT=$(shell git rev-parse --short=7 HEAD)
 OPERATOR_VERSION=$(VERSION_MAJOR).$(VERSION_MINOR).$(COMMIT_NUMBER)-$(CURRENT_COMMIT)
 KUBECTL ?= kubectl
 
+# for boilerplate
+OPERATOR_NAME=route-monitor-operator
```
Comment (2uasimojo) [inline Makefile:10]: FWIW, boilerplate is set up to [discover](https://github.com/openshift/boilerplate/blob/69bf49e355dd9fe677a6c2d7a295949f6a2998e7/boilerplate/openshift/golang-osd-operator/project.mk#L2) this value from a `config/config.go` file ([example](https://github.com/openshift/aws-account-operator/blob/master/config/config.go)).

(I'll be the first to concede that's pretty janky, especially as I don't recall seeing any project where that config.go is actually used in the operator code. Something to polish later.)

**Review by 2uasimojo — COMMENTED** [review]
All these changes seem cool to me.

File: controllers/routemonitor/routemonitor_supplement.go (hunk 1 above)
Comment (RiRa12621) [reply controllers/routemonitor/routemonitor_supplement.go:57]: ack

**Comment by 2uasimojo** [comment]
> /cancel approve

Is it spelled `/approve cancel`?

**Comment by RiRa12621** [comment]
> > /cancel approve
> 
> Is it spelled `/approve cancel`?

yes, you're right. whoopsie
/approve cancel

**Comment by NautiluX** [comment]
/close
superseded by PR #82

**Pull request closed without merging**